Formalisation(shacl)/Core/PiecesShape/Dataset.ttl

```diff
@@ -171,7 +171,6 @@
 
 <http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#legal-basis> sh:description
     "The legal basis used to justify processing of personal data"@en;
-  sh:minCount 1;
```

Comment:
This is not conform the excel version from data team.

Reply:
Disregard, the introduction of different levels of "openness" require a solution on a different level.